scripts/circleci/release/installScalyrAgentV2.sh

```diff
@@ -378,18 +393,28 @@ if [[ $REPO_TYPE == "yum" ]]; then
   run_command "yum remove -y scalyr-repo";
   run_command "yum remove -y scalyr-repo-bootstrap";
 
-  lecho "  Installing Scalyr yum repository configuration files.";
+  if $USE_BOOTSTRAP_PACKAGES; then
+    lecho "  Installing Scalyr yum repository configuration files.";
 
-  # Install the bootstrap rpm for the repository configuration.  This isn't
-  # signed since it is the first package, so disable the gpg check.
-  run_command "yum install --nogpgcheck -y $bootstrap_rpm";
+    # Install the bootstrap rpm for the repository configuration.  This isn't
+    # signed since it is the first package, so disable the gpg check.
+    run_command "yum install --nogpgcheck -y $bootstrap_rpm";
 
-  lecho "  Updating Scalyr yum repository configuration files using repository.";
-  # Now, use the repository to update the repository configuration.  This
-  # removes the old bootstrap package and allows the repository to distribute
-  # updates to the repository configuration itself.  This is the cleanest
-  # way we found to make sure the repository configuration can be updated.
-  run_command "yum install -y scalyr-repo";
+    lecho "  Updating Scalyr yum repository configuration files using repository.";
+    # Now, use the repository to update the repository configuration.  This
+    # removes the old bootstrap package and allows the repository to distribute
+    # updates to the repository configuration itself.  This is the cleanest
+    # way we found to make sure the repository configuration can be updated.
+    run_command "yum install -y scalyr-repo";
+  else
+      # Add tool that may be needed to add repository.
+      run_command "yum install -y yum-utils"
+      echo "Adding the public key."
+
+      run_command "rpm --import https://${KEYSERVER_URL}/pks/lookup?op=get&search=0x${PUBLIC_KEY_FINGERPRINT}"
+      echo "Adding the scalyr agent repository."
+      run_command "yum-config-manager --add-repo ${REPOSITORY_URL}/yum/binaries/noarch"
```

Comment:
I take it `yum-config-manager` is fairly standard?  We aren't going to have problems on older installations?

Reply:
It should be a very common tool, but I've also managed how to add repo directly, without any tools.